src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
+import java.util.*;
+
 public class Duke {
+    public static String partition = "------------------------------------------------------------";
```

Comment:
Consider changing 'partition' to a final variable as it does not change, as well as fully capitalising the variable name to be 'PARTITION'